sdk/framework/openapi.go

```diff
@@ -773,67 +1362,3 @@ func cleanResponse(resp *logical.Response) *cleanedResponse {
 		Headers:  resp.Headers,
 	}
 }
-
-// CreateOperationIDs generates unique operationIds for all paths/methods.
-// The transform will convert path/method into camelcase. e.g.:
-//
-// /sys/tools/random/{urlbytes} -> postSysToolsRandomUrlbytes
-//
-// In the unlikely case of a duplicate ids, a numeric suffix is added:
-//
-//	postSysToolsRandomUrlbytes_2
-//
-// An optional user-provided suffix ("context") may also be appended.
-func (d *OASDocument) CreateOperationIDs(context string) {
```

Comment:
dont think this is used anywhere but could be breaking change

Reply:
I don't think it's used anywhere, at least according to public github search. This should not have been exported.